packages/core/test-helpers/core-test-helpers-kbn-server/src/create_serverless_root.ts

```diff
@@ -28,6 +28,8 @@ export interface TestServerlessUtils {
 
 const ES_BASE_PATH_DIR = Path.join(REPO_ROOT, '.es/es_test_serverless');
 
+const projectType: ServerlessProjectType = 'es';
```

Comment:
`projectType` is required for ES startup in order provision project-specific roles only. I guess this tests are project agnostic and this change is ok?